pkg/skuba/actions/cluster/status/status.go

```diff
@@ -34,7 +34,24 @@ func Status(client clientset.Interface) error {
 		return errors.Wrap(err, "could not retrieve node list")
 	}
 
-	outputFormat := "custom-columns=NAME:.metadata.name,OS-IMAGE:.status.nodeInfo.osImage,KERNEL-VERSION:.status.nodeInfo.kernelVersion,KUBELET-VERSION:.status.nodeInfo.kubeletVersion,CONTAINER-RUNTIME:.status.nodeInfo.containerRuntimeVersion,HAS-UPDATES:.metadata.annotations.caasp\\.suse\\.com/has-updates,HAS-DISRUPTIVE-UPDATES:.metadata.annotations.caasp\\.suse\\.com/has-disruptive-updates,CAASP-RELEASE-VERSION:.metadata.annotations.caasp\\.suse\\.com/caasp-release-version"
+	for _, item := range nodeList.Items {
+		if _, ok := item.Labels["node-role.kubernetes.io/master"]; ok {
```

Comment:
with this logic it assumes that whatever node that is not labeled as `node-role.kubernetes.io/master"` is a worker, is this a safe assumption? can be nodes that are not labeled?

Reply:
Good question.
The answer would be:
```
kubectl get nodes
NAME       STATUS   ROLES    AGE   VERSION
master-0   Ready    master   18h   v1.16.2
worker-0   Ready    <none>   18h   v1.16.2
worker-1   Ready    <none>   18h   v1.16.2
```
You can see what get role here with `kubectl get nodes -o json`

Reply:
:thinking: If I understand https://github.com/kubernetes/community/pull/911 right, then this should be a find assumption and one we can update later. I'm not worried that this will have much of a lasting hard to fix problem. if there becomes another node role we want to care about, we can update it then.

Reply:
The answer here depends on whether we want to reflect "what did skuba say" or "what is the node role".

Skuba only allows two roles during bootstrap, but that's misleading, IMO.  Nodes can have arbitrary roles.  The `master` role is treated specially when scheduling jobs, but a customer site can put an arbitrary value in the node using the label format `node-role.kubernetes.io/<role>`.  The "right" behavior if we're trying to stay close to upstream is to use the kubectl output format as referenced in https://github.com/SUSE/skuba/pull/928#discussion_r370202922.  We should therefore emit `<none>` if no role is assigned, otherwise emit the contents of that label.  Documentation can explain that worker nodes default to `<none>` unless a site-specific role is assigned to that node.